File: python/hamming/hamming.py
def distance(strand_a, strand_b):
    if ( not len(strand_a) == len(strand_b)):
        raise ValueError("Issue")
    count = 0
    for i in range(len(strand_a)):
        if not strand_a[i] == strand_b[i]:
            count += 1
    return count

File: javascript/hamming/hamming.js
export const compute = (strandA, strandB) => {

  let count = 0;
  if (strandA.length == 0 && strandB.length == 0) return 0;
  if (strandA.length == 0) throw new Error("left strand must not be empty");
  if (strandB.length == 0) throw new Error("right strand must not be empty");
  if (strandA.length !== strandB.length) throw new Error("left and right strands must be of equal length");

  for(let i = 0; i < strandA.length; i++){
    if (strandA[i] !== strandB[i]) count += 1;
  }
  return count;
};